envision/client.py

```diff
@@ -160,6 +160,7 @@ def on_close(ws):
 
         def on_error(ws, error):
             self._log.warning("Unable to connect to Envision")
+            self._log.debug(error)
```

Comment:
Remove this? Or log the error as an error?

Reply:
I will upgrade the warning to an error and merge the two logs as an error. The current without the error being included is misleading.